platforms/ios/lib/WysiwygComposer/Sources/WysiwygComposer/Extensions/ComposerModel.swift

```diff
@@ -55,4 +55,11 @@ extension ComposerModel {
 
         return update
     }
+
+    /// Returns currently reversed (active) actions on the composer model.
```

Comment:
> currently reversed (active) actions

Outside the scope of this change, but I find it a bit confusing that 'reversed' is equivalent to 'active'. Should we consider separating the concepts of toggling ('reversing') the state from the state itself (i.e. active/inactive)?

Reply:
It can be a bit confusing, but IIRC the reasoning behind this is that the ComposerAction refers to calling the associated function as well. e.g. if bold is reversed, calling `bold` will actually `unbold`. If we flagged this as `active` instead, one could assume that calling `bold` would do nothing since it's already active.